src/tree/containers/ContainersTreeItem.ts

```diff
@@ -51,14 +55,18 @@ export class ContainersTreeItem extends LocalRootTreeItemBase<DockerContainer, C
         return this.groupBySetting === 'None' ? 'container' : 'container group';
     }
 
-    public async getItems(context: IActionContext): Promise<DockerContainer[]> {
-        const results = await ext.dockerClient.getContainers(context);
+    public async getItems(context: IActionContext): Promise<DockerContainerInfo[]> {
+        const rawResults = await ext.dockerClient.getContainers(context);
+        const showFiles = (await ext.dockerContextManager.getCurrentContext())?.Type === 'moby';
```

Comment:
Depending on how ECS works, it might be better to specifically exclude ACI, rather than specifically include Moby. What do you think?

Reply:
(For context, we're trying to make it so that at least in theory, an ECS context "just works" even though we don't really know anything about it yet)

Reply:
Can do.